src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Extensions/DbContextTestExtensions.cs

```diff
@@ -0,0 +1,32 @@
+using System;
```

Comment:
`#nullable enable` is missing here

Reply:
Nice one.  Added in and also to DbContextTransactionExtensions as well.